docs/src/architecture/assets.md

```diff
@@ -0,0 +1,53 @@
+# Assets
+The Miden Rollup aims to provide a reliable protocol and software for users to exchange assets. In Miden, users can create and trade arbitrary fungible and non-fungible assets. 
+
```

Comment:
The first sentence sounds odd to me - I'd probably delete it.

Reply:
Isn't everything about assets?